src/react-apps/applications/runtime/src/features/form/dynamics/sagas/fetch/index.ts

```diff
@@ -0,0 +1,23 @@
+import { SagaIterator } from 'redux-saga';
+import { takeLatest, call } from 'redux-saga/effects';
+
+import FormDynamicsActions from '../../actions';
+import * as FormDynamicsActionTypes from '../../actions/types';
+import { IFetchServiceConfig } from '../../actions/fetch';
+// import { get } from 'Shared/utils/networking';
+
+import { serviceConfig } from './testData';
+
+function* fetchDynamicsSaga({ url }: IFetchServiceConfig): SagaIterator {
+  try {
+    // const serviceConfig = yield call(get, url);
```

Comment:
why is this commented out?

Reply:
I'm using offline data (from the testData-file, which is just a copy paste of the json returned from the api), and this will be uncommented when merging to master.

Just because it's easier and faster to develop without having all of altinn studio running in the background.

Reply:
cool 😎